docs/spelling_wordlist.txt

```diff
@@ -33,6 +33,7 @@ deserialized
 deserializers
 deserializes
 deserializing
+e
```

Comment:
That is curious because we already have "i.e." in the master branch. Maybe something changed on Travis...